itch_dl/cli.py

```diff
@@ -41,6 +41,8 @@ def parse_args() -> argparse.Namespace:
                         help="filter downloaded files with a Python regex (unmatched files are skipped)")
     parser.add_argument("--verbose", action="store_true",
                         help="print verbose logs")
+    parser.add_argument("--hh-export", action="store_true",
+                        help="Export Homebrew Hub compliant entries")
 
```

Comment:
- Let's move this above `--verbose` so that it's always the last option.
- Can you add this to [Settings](https://github.com/DragoonAethis/itch-dl/blob/main/itch_dl/config.py#L26) as well?

```suggestion
                        help="filter downloaded files with a Python regex (unmatched files are skipped)")
    parser.add_argument("--hh-export", action="store_true",
                        help="Export Homebrew Hub compliant entries")
    parser.add_argument("--verbose", action="store_true",
                        help="print verbose logs")

```

Reply:
Done and added to Settings